ygen/codegen.go

```diff
@@ -102,6 +102,14 @@ type GoOpts struct {
 	// GenerateRenameMethod specifies whether methods for renaming list entries
 	// should be generated in the output Go code.
 	GenerateRenameMethod bool
+	// AddAnnotationFields specifies whether annotation fields should be added to
+	// the generated structs. When set to true, a metadata field is added for each
+	// struct, and for each field of each struct. Metadata field's names are
+	// prefixed by the string specified in the AnnotationPrefix argument.
+	AddAnnotationFields bool
+	// AnnotationPrefix specifies the string which is prefixed to the name of
+	// annotation fields. It defaults to Λ.
```

Comment:
The part that says what it defaults to is likely to get out-of-sync with the default value in the annotation_prefix flag.

We can either remove that note from here or define a const like DefaultAnnotationFieldPrefix and reference that both from here and from the flag.

Reply:
Yes, sorry -- this was the pattern that I used elsewhere, not sure why I didn't here. Fixing this.